src/pages/public_registration/LogPanel.tsx

```diff
@@ -41,6 +43,68 @@ export const LogPanel = ({ tickets, registration }: LogPanelProps) => {
     cacheTime: 1_800_000, // 30 minutes
   });
 
+  const logs: LogList = [];
```

Comment:
Nesten ikke pågjort med egen type, men går jo for det samme :D 
```suggestion
  const logs: LogItem[] = [];
```